docs/wiki/Native-BTC-Wallet-Recovery-and-Rescanning.md

```diff
@@ -1,38 +1,66 @@
+If the native BTC wallet is corrupted, the dexc.log may have a message containing "wallet not found", such as:
+
+> [ERR] CORE: Unable to connect to **btc** wallet (start and sync wallets BEFORE starting dex!): **connectWallet: failed to connect btc wallet: connect failure: wallet not found**
 
-> [ERR] CORE: Unable to connect to **btc** wallet (start and sync wallets BEFORE starting dex!): **connectWallet: wallet not found**
+Note **btc** in the above message, and **wallet not found**.  This may also say something about db corruption, or similar errors, which may also be addressed as follows.
 
-Note **btc** in the above message, and **wallet not found**.  This may also say something about db corruption or similar errors, which may also be addressed as follows.
+In this case, the Wallets page (`/wallets`) will still show the wallet since it was previously configured, just that it is not synchronized and cannot be unlocked or connected.  It will look like this:
 
-In this case, the Wallets page (/wallets) will still show the wallet since it was previously configured, just that it is not synchronized and cannot be unlocked:
+<img alt="BTC Wallet View" src="./images/unloaded-btc-wallet.png" width="300"/>
 
-![wallets view](https://user-images.githubusercontent.com/9373513/149597739-06318c29-f6be-4837-bb6f-2055971be8cf.png)
+To reinitialize the wallet, [open the wallet settings](#opening-the-btc-wallet-settings) and click the `Submit` button.
 
-This may be solvable by simply clicking the gears to open the settings box, and then click Submit:
+If successful you will see a message like:
 
-![wallet settings](https://user-images.githubusercontent.com/9373513/149597817-abbcbf5f-d05f-4b48-996f-485030d2a74c.png)
+<img alt="BTC Wallet Reconfigured Success" src="./images/wallet-reconfigured-success.png" width="300"/>
 
-The dexc.log should say:
+and the dexc.log should say:
 
-> [INF] CORE: Initializing a built-in btc wallet
+> [INF] CORE: Initializing a btc wallet
 
 The wallet will begin re-synchronizing.  This may take several minutes.
 
-If this process results in errors, it may be necessary to go to the folder containing the wallet data and **delete the wallet.db so it may be regenerated** from your application seed.  First SHUTDOWN DEX. Next, for mainnet, on Linux, this would be in the folder `~/.dexc/mainnet/assetdb/btc/mainnet/`.  There may be a wallet.db file in this folder, which you should rename wallet.db.bak.  Now startup and login to DEX.  Go to the Wallets page to see wallet status, which should re-synchronize.  Check notifications and logs for any errors.
+**IF** this process results in errors, it may be necessary to go to the folder containing the wallet data and **delete the wallet.db so it may be regenerated** from your application seed.  To do this:
+
+1. Shutdown DEX.
+2. Open the folder for the wallet `network` (either `mainnet`, `testnet`, `regtest`).  This would be:
```

Comment:
You're not using `network` below as a placeholder, nor should you. Just say something like 

`Open the wallet directory. For mainnet, this would be:`

It's less confusing for casual users and anybody using testnet or simnet will know what to do.